### PR TITLE
Specifying workspace resolver

### PR DESCRIPTION
**Pull request by Darkros1245**
Fixes an issue with the workspaces by specifying the resolver

### CODE REVIEW

File: client/src/main.rs [hunk 1]
```diff
@@ -21,6 +21,7 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
     Ok(())
 }
 
+#[allow(dead_code)]
```
Comment (MJHC) [inline client/src/main.rs:24]: this looks suspicious

**Review by MJHC — APPROVED** [review]
lgtm

File: client/src/main.rs (hunk 1 above)
Comment (Darkros1245) [reply client/src/main.rs:24]: nah

**Comment by Laursen79** [comment]
Why is this pull request called "Fix"? What does it fix?

**Review by Laursen79 — APPROVED** [review]

**Pull request merged**